examples/build_sample_experiment.py

```diff
@@ -149,14 +167,31 @@ def write_experiment_json(path, fov_count, hyb_dimensions, aux_name_to_dimension
         Dictionary mapping dimension name to dimension size for the hybridization image.
     aux_name_to_dimensions : Mapping[str, Mapping[str, int]]
         Dictionary mapping the auxiliary image type to dictionaries, which map from dimension name to dimension size.
+    hyb_image_fetcher : Optional[ImageFetcher]
```

Comment:
This and the two following comments could use a bit more verbosity -- What is an ImageFetcher and why would a user pass it? 

Other things I'm wondering here -- it looks like ImageFetcher looks for single images, but I want to build an experiment! so, how does this work?

Reply:
Added more prose.